integration/bin/alluxio-master-mesos.sh

```diff
@@ -12,8 +12,18 @@
 
 
 SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd)"
+
+if [[ -z "${ALLUXIO_RAM_FOLDER}" ]]; then
```

Comment:
could we avoid the duplication by putting it in a separate file and sourcing it here and from the worker script?